### PR TITLE
Begin puppet config

### PR DESCRIPTION
**Pull request by jdelnano**

### CODE REVIEW

File: manifests/default.pp [hunk 1]
```diff
@@ -1,4 +1,14 @@
...
+  ensure       => latest,
+  manage_repos => true,
+  fpm          => true,
+  dev          => true,
+  composer     => true,
+  pear         => true,
+  phpunit      => false,
+}
+
+class { '::mysql::server':
```
Comment (jdelnano) [inline manifests/default.pp]: @ddelnano here is the class i tried to write.

File: manifests/default.pp [hunk 2]
```diff
@@ -1,4 +1,14 @@
...
+  root_password          => 'strongpassword',
+  remove_default_accounts => true
+}
```
Comment (jdelnano) [inline manifests/default.pp]: @ddelnano here is the class i tried to write.

**Comment by ddelnano** [comment]
Yea but what is the error?

File: manifests/default.pp (hunk 2 above)
Comment (jdelnano) [reply manifests/default.pp]: I took our the ::mysql::server class, and even when I run the command for puppet (the one at the bottom of the Vagrantfile), it gives me this error.

screen shot 2017-09-15 at 10 59 20 am

Comment (ddelnano) [reply manifests/default.pp]: what is the command you are running puppet with?

**Pull request closed without merging**